addons/im_livechat/static/src/core/public_web/discuss_channel_model_patch.js

```diff
@@ -13,5 +13,8 @@ const discussChannelPatch = {
             },
         });
     },
+    get memberListTypes() {
+        return [...super.memberListTypes, "livechat"];
+    },
```

Comment:
I don't understand why this would be in `public_web` when the rest, including usage of the field, is in `common`.

Reply:
It moved from `public_web/thread_model_patch` to `public_web/discuss_channel_model_patch`. I guess it wasn't in the correct bundle initially